task/build-maven-zip/0.1/README.md

```diff
@@ -0,0 +1,29 @@
+# Build-maven-zip task
+
+Build-maven-zip task builds prefetched maven artifacts into a OCI-artifact with zip bundle  and pushes the OCI-artifact into container registry.
+In addition it generates a SBOM file, injects the SBOM file into final container image and pushes the SBOM file as separate image using cosign tool.
+Note that this task needs the output of prefetch-dependencies task. If it is not activated, there will not be any output from this task.
+
+## Parameters
+
+| name                 | description                                                            | default value    | required |
+| -------------------- | ---------------------------------------------------------------------- | ---------------- | -------- |
+| IMAGE                | Reference of the OCI-Artifact this build task will produce.            |                  | true     |
+| PREFETCH_ROOT        | The root directory of the artifacts in the prefetched directory.       | maven-repository | false    |
+| BUNDLE_NAME          | The zip bundle name of archived artifacts.                             | maven-repository | false    |
```

Comment:
Not any prefer from my side. Ok to change.

Reply:
Changed in new commit

Reply:
> But the default name should not include .zip there because it is handled in the task logic.

Ah, that's what I meant - that the parameter should be the whole filename, not just the part before the task adds `.zip`. It's just clearer what the parameter means that way